src/EnergyPlus/EcoRoofManager.cc

```diff
@@ -413,15 +414,15 @@ namespace EcoRoofManager {
                 Gammah = std::pow(1.0 - 5.0 * Rib, -0.5);
             }
 
-            if (RoughSurf == DataSurfaces::SurfaceRoughness::VerySmooth) {
+            if (RoughSurf == Material::SurfaceRoughness::VerySmooth) {
```

Comment:
Can replace this with a `std::array` and lookup.

Reply:
yeah, that's better. I'll make the Zog values into an array and look it up using roughness